.travis.yml

```diff
@@ -250,28 +285,50 @@ jobs:
       before_install:
         - sudo bash ops/travis/system-testing/build_and_run before_install
       install:
-        - bash ops/travis/system-testing/build_and_run install
+        - unbuffer bash ops/travis/system-testing/build_and_run install
       before_script:
-        - bash ops/travis/system-testing/build_and_run before_script
+        - unbuffer bash ops/travis/system-testing/build_and_run before_script
       script:
-        - bash ops/travis/system-testing/build_and_run script
+        - unbuffer bash ops/travis/system-testing/build_and_run script
       after_success:
-        - bash ops/travis/system-testing/build_and_run after_success
+        - unbuffer bash ops/travis/system-testing/build_and_run after_success
       after_failure:
-        - bash ops/travis/system-testing/build_and_run after_failure
+        - unbuffer bash ops/travis/system-testing/build_and_run after_failure
+
+    - stage: deployment
+      name: master
+      if: branch = master
+      git:
+        depth: false
+      script: echo "Deploy master"
       deploy:
         - provider: script
-          skip_cleanup: true
-          script: bash ops/travis/staging/deploy
+          script: bash ops/travis/deploy/master
           on:
             branch: master
+    - stage: deployment
+      name: staging
+      if: branch = staging
+      git:
+        depth: false
+      script: echo "Deploy staging"
+      deploy:
         - provider: script
-          skip_cleanup: true
-          script: ops/travis/production/deploy
+          script: bash ops/travis/deploy/staging
```

Comment:
no need to prefix bash with ``unbuffer``

Reply:
I do not really understand what you mean with this comment.
I added unbuffer to get the color of the tests back in travis logs to circumvent a bug from travis. It's possible they fixed it in the meantime?

Reply:
I mean exactly that :-) ... do you need colors here?

Reply:
ah ok... was missing a ? at the end I guess.. it's a bit late ;)
done